gravity.sh

```diff
@@ -524,7 +524,13 @@ parseList() {
 
   # Create a temporary file for the sed magic instead of using "${target}" directly
   # this allows to split the sed commands to improve readability
+  # we use a file handle here and remove the temporary file immediately so the content will be deleted in any case
+  # when the script stops
   temp_file="$(mktemp -p "/tmp" --suffix=".gravity")"
+  exec 3>"$temp_file"
+  rm "${temp_file}"
+  temp_file="/proc/$$/fd/3"
```

Comment:
If you need to keep `temp_file` for use in other places inside the script then please use two different variable names instead of shadowing an existing variable name.

```suggestion
  temp_file_base="$(mktemp -p "/tmp" --suffix=".gravity")"
  exec 3>"$temp_file_base"
  rm "${temp_file_base}"
  temp_file="/proc/$$/fd/3"
```